docker/sftp/data/incoming/SSCS_Extract_Delta_No_Parties_CaseUpdated_2024-05-10-23-34-59.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" standalone="yes"?>
```

Comment:
do we need this in source control?

Reply:
Its in the existing examples, and on the xml we find on azure